FlowersStore/Utils/Layout/UIView+Constraints.swift

```diff
@@ -0,0 +1,212 @@
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func leadingAnchor(equalTo anchor: NSLayoutXAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = leadingAnchor.constraint(equalTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func leadingAnchorToSuperview(constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        guard let superview = superview else { fatalError("View doesn't have a superview") }
+        let constraint = leadingAnchor.constraint(equalTo: superview.leadingAnchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func trailingAnchor(equalTo anchor: NSLayoutXAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = trailingAnchor.constraint(equalTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func trailingAnchor(greaterThanOrEqualTo anchor: NSLayoutXAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = trailingAnchor.constraint(greaterThanOrEqualTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func trailingAnchor(lessThanOrEqualTo anchor: NSLayoutXAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = trailingAnchor.constraint(lessThanOrEqualTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func trailingAnchorToSuperview(constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        guard let superview = superview else { fatalError("View doesn't have a superview") }
+        let constraint = trailingAnchor.constraint(equalTo: superview.trailingAnchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func heightAnchor(equalTo height: CGFloat, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = heightAnchor.constraint(equalToConstant: height)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func heightAnchor(equalTo nsLayoutDimension: NSLayoutDimension, multiplier: CGFloat = 1.0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
```

Comment:
<!--
  0 failure: 
  1 warning:  Function 'heightA...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Utils/Layout/UIView+Constraints.swift;
  Line: 145;
-->

- :warning: Function 'heightAnchor(equalTo:multiplier:priority:)' is unused